src/components/flowchart/flowchart.js

```diff
@@ -485,7 +488,11 @@ export class FlowChart extends Component {
    * @param {Object} node Datum for a single node
    */
   handleNodeMouseOver = (event, node) => {
-    this.props.onToggleNodeHovered(node.id);
+    if (node.type === 'modularPipeline') {
```

Comment:
hey @rashidakanchwala why do we need this? I thought this can be done via css?

Reply:
We need to add the --active attribute to modular pipelines and that only happens if you update the redux state.

Reply:
u can try it if possible via css

Reply:
https://github.com/user-attachments/assets/19177b08-c37e-4199-9773-27fe5585fb92


Am I missing something??  it's working fine without adding the active attribute here. If the video looks fine to you I'll remove this extra changes

Reply:
it's the other way round. hovering over the flowchart and see the effects on flowchart/nodelist. if you hover over the Modular Pipelines.

Reply:
ah I see, got it now. Let me test again